manifests/init.pp

```diff
@@ -15,17 +15,24 @@
   String $ldap_bind_pass,
   String $ldap_base_dn,
   String $ldap_group_dn,
+  String $smtp_host,
+  String $smtp_user,
+  String $smtp_pass,
+  String $smtp_mail_from,
   String $log_level,
   String $pip_args,
   Variant[Enum['system', 'pypy'], String[1]] $python_version,
   Integer $concurrency,
+  Integer $smtp_port,
```

Comment:
Has no default to `common.yaml`.
I'd be tempted to add one for this because it's likely to be 90% the same port… but to have the other params default to `undef` rather than empty string.